### ISSUE TITLE
Suggestion from KingCosmic

### ISSUE THREAD

**Issue by jdotrjs** [phaser-discord/administration#28]
Suggested by KingCosmic#2713 / 152186808732483584, on Friday, April 13, 2018, 22:56 UTC:
>1. Add a updates channel for updates to phaser.

2. Maybe make different categories for phaser 3 and phaser ce

**Comment by 16patsle** [comment]
Splitting up v2 and v3 would make sense in that you wouldn't need to ask people which of them they're using, which I noticed is often necessary, especially in #beginners.

The updates channel could be a good idea if we could automate it. I know there is a github channel in Slack which has webhook integration with the repo, but I guess it would be overkill to have the same in Discord.

Having a webhook that triggers on a new release (which you can do, I checked) would be an option, but in that case we'd have to ask Rich if he's willing to add a webhook to the repo. If we could do that, it would be pretty neat.

**Comment by jdotrjs** [comment]
the CE thing is mostly a question of "are people actually good at self selecting into channels?" I've had folks send v2 questions into v3 after I've explicitly let them know that they shouldn't do that as it just wastes time so I'm maybe a bit too cynical :stuck_out_tongue: 

No real objection to a channel that hooks phaser updates to a channel but I do think it should be sent to a specific channel instead of going to one of the main ones for chat. One consideration is that if we go that route and decide it's a bad idea we'll have to involve rich to disable it (I think?) so that might set a higher bar for "it's worth it to try" since it's not as low  effort as other stuff

**Comment by 16patsle** [comment]
If we were to divide it into v2 and v3, we should probably try to limit the channel amount, maybe to just #general, #v2 and #v3, but then we wouldn't have a separate channel for beginners, unless we want to have #v2-beginners and #v3-beginners as well, but I feel like that would be overkill.

We could always disable the phaser update channel by removing the webhook on the discord side, but you're right that removing it properly would require Rich removing it from the repo. I agree that we should send it to it's own channel (maybe combining it into an #announcements channel in the future, so we can post other stuff there as well?).

**Comment by jdotrjs** [comment]
Oh hm. dropping #beginners for #phaser-v2 seems like a good move. It's totally unclear what beginner even means and at times it feels kind of derisive. I'd be into that.

**Comment by 16patsle** [comment]
Yeah, I feel like separating v2 and v3 is more relevant than beginner/not-beginner, and as you say, the line between beginner/not-beginner is pretty fuzzy. Would the most logical thing be to just rename #beginner to #phaser-2? I feel like much of the discussion in there is v2 already anyways.

**Comment by 16patsle** [comment]
The current plan is to change the #beginners channel over to #phaser-ce (or a similar name).
Considering making a channel for #updates, where we can post new issues of the newsletter and new releases of Phaser. Thoughts?
